.github/workflows/configlet.yml

```diff
@@ -0,0 +1,16 @@
+name: Configlet CI
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  configlet:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
```

Comment:
Any reason why we use a SHA here, rather than a version tag?

Reply:
SHAs are more secure than tags, as someone can untag and retag at a different sha.

Reply:
More context here: https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions